Tribler/community/multichain/database.py

```diff
@@ -288,7 +288,11 @@ def check_database(self, database_version):
         assert int(database_version) >= 0
         database_version = int(database_version)
 
-        if database_version < 1:
+        if database_version != LATEST_DB_VERSION:
```

Comment:
Nice catch, will include this in the new PR. I remember Pim saying something about the version number working oddly for the MC database. (Something to do with it being 0 on initialized environments sometimes?) The only purpose the != serves is to screw things up when someone uses a new database schema with old code, but we shouldn't expect that to work anyway ^.^